transactions/flows/flowMap.ts

```diff
@@ -76,4 +84,15 @@ export const TRANSACTION_FLOW_MAP: {
     tx: async (params) => clmClaimRewardsTx(params),
     validRetry: async (params) => validateClmClaimRewardsRetryTx(params),
   },
+  [TransactionFlowType.STAKE_CANTO_TX]: {
+    tx: async (params: StakingTransactionParams) => stakingTx(params),
+    validRetry: async (params: StakingTransactionParams) =>
+      NO_ERROR({ error: false }),
+  },
+  [TransactionFlowType.VOTE_TX]: {
+    tx: async (params: ProposalVoteTxParams) => proposalVoteTx(params),
+    validRetry: async (params: ProposalVoteTxParams) =>
```

Comment:
same thing. We can actually verify and retry. As long as the proposal is still active let's allow a retry here